pom.xml

```diff
@@ -58,6 +58,7 @@
         <cypherdsl.version>2020.1.4</cypherdsl.version>
         <node.version>v12.16.0</node.version>
         <npm.version>6.13.7</npm.version>
+        <java.version>11</java.version>

```

Comment:
We cannot do that because we still need to support Java 8.
Please look at:
https://spark.apache.org/docs/3.0.0/

_Spark runs on Java 8/11_